### PR TITLE
Added date, replaced spaces by tabs and added break lines between functions

### PR DESCRIPTION
**Pull request by BrianSantivanez**
Issue link: https://github.com/softlayer/githubio_source/issues/528

Observations: The error of the date was that month and day were exchanged, break lines between functions were added to improve readability and the type of code box was changed to match with other examples

### CODE REVIEW

File: content/go/notification_ocurrent_events.go.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Managing Notification Occurrence Events services"
 description: "How to interact with the Notification Ocurrence Events services. The SoftLayer_Notification_Occurrence_Event service represents all events with potential to cause a disruption in service.."
-date: "2021-18-01"
+date: "2022-01-13"
```
Comment (edsonarios) [inline content/go/notification_ocurrent_events.go.md:4]: Maybe the date should not be updated to today, just fixed

Comment (BrianSantivanez) [reply content/go/notification_ocurrent_events.go.md:4]: The date was only fixed

File: content/go/notification_ocurrent_events.go.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Managing Notification Occurrence Events services"
 description: "How to interact with the Notification Ocurrence Events services. The SoftLayer_Notification_Occurrence_Event service represents all events with potential to cause a disruption in service.."
```
[the remark this answers is not in the document]
Comment (BrianSantivanez) [reply content/go/notification_ocurrent_events.go.md:3]: The second point in the title was removed

File: content/go/notification_ocurrent_events.go.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-date: "2021-18-01"
+date: "2021-01-18"
 classes: 
     - "SoftLayer_Notification_Occurrence_Event"
```
[the remark this answers is not in the document]
Comment (BrianSantivanez) [reply content/go/notification_ocurrent_events.go.md:6]: The tag was changed by "Notification"

**Review by edsonarios — APPROVED** [review]
Looks good to me

**Review by allmightyspiff — APPROVED** [review]
Looks good, thanks

**Pull request merged**